airflow/config_templates/config.yml

```diff
@@ -2379,7 +2379,7 @@ elasticsearch:
 elasticsearch_configs:
   description: ~
   options:
-    use_ssl:
```

Comment:
Why this change?

Reply:
In elasticsearch 7, `use_ssl` is an accepted parameter when constructing `ElasticSearch` client. See the following source code : 

https://github.com/elastic/elasticsearch-py/blob/7.14/elasticsearch/client/__init__.py#L113

However, in elasticsearch 8, it no longer accepts `use_ssl` parameter. See the following source code: 

https://github.com/elastic/elasticsearch-py/blob/8.9/elasticsearch/_sync/client/__init__.py#L129

Therefore to make the testsuite compile with the ES8 , I use `http_compress` as the argument (which is one of the accepted arguments for constructing ES client